spark/src/main/resources/error/delta-error-classes.json

```diff
@@ -2371,6 +2371,19 @@
     ],
     "sqlState" : "XXKDS"
   },
+  "DELTA_TABLE_INVALID_REDIRECT_STATE_TRANSITION" : {
```

Comment:
Could you add link to RFC in PR description?

Reply:
Done